File: solutions/2654-count-the-number-of-vowel-strings-in-range/solution.py
class Solution(object):
    def vowelStrings(self, words, left, right):
        """
        :type words: List[str]
        :type left: int
        :type right: int
        :rtype: int
        """
        c=0
        for i in range(left,right+1):
            if words[i][0] in "aeiou" and words[i][-1] in "aeiou":
                c+=1
        return c

File: solutions/2654-count-the-number-of-vowel-strings-in-range/solution.java
class Solution {
    public int vowelStrings(String[] words, int left, int right) {
        int count=0;
        for(int i=left;i<=right;i++)
        {
            char a=words[i].charAt(0);
            char b=words[i].charAt(words[i].length()-1);
            if((a+"").matches("[aeiou]+") && (b+"").matches("[aeiou]+"))
                count++;
        }
        return count;
    }
}
